UIKitHomework/UIKitHomework/MainViewController.swift

```diff
@@ -0,0 +1,217 @@
+//
+//  MainViewController.swift
+//  UIKitHomework
+//
+//  Created by MacBook Эдуард on 05.07.2021.
+//
+
+import UIKit
+
+class MainViewController: UIViewController {
```

Comment:
```suggestion
final class MainViewController: UIViewController {
```
нейминг не совсем подходит, лучше может какой то TextEditorViewController